server/wordi/src/main/java/com/pm/wordi/service/MentorService.java

```diff
@@ -128,4 +128,24 @@ public MentoringProfileRes getMentoringProfile(Long mentorId) {
                 .map(MentoringProfileRes::new)
                 .orElseThrow(() -> new NoExistMentoringProfileException("해당 멘토 프로필이 존재하지 않습니다."));
     }
+
+    @Transactional
+    public void updateProfileImage(Long userId, @Nullable MultipartFile profileImage) {
+        Mentor mentor = mentorRepository.findByUserIdAndStatus(userId, ACTIVE)
+                .orElseThrow(() -> new NoExistMentorException("해당 회원의 멘토 정보가 존재하지 않습니다."));
+
+        // 프로필 이미지 미첨부
+        if (profileImage.isEmpty()) {
+            mentor.updateProfileImage(null);
+        }
+
+        // 프로필 이미지 첨부
+        if (!profileImage.isEmpty()) {
+            String s3SaveFileName = awsS3Service.uploadBucket(profileImage);
+            String fileFullPath = "https://" + CLOUD_FRONT_DOMAIN_NAME + "/" + s3SaveFileName;
+            mentor.updateProfileImage(fileFullPath);
+        }

```

Comment:
문자열 덧셈에 Stringbuffer를 써보시는건 어떠실까요?
```suggestion
        // 프로필 이미지 첨부
        if (!profileImage.isEmpty()) {
            StringBuffer sb = new StringBuffer();
            String s3SaveFileName = awsS3Service.uploadBucket(profileImage);
            String fileFullPath =sb.append("https://")
                        .append(CLOUD_FRONT_DOMAIN_NAME).append("/").append(s3SaveFileName).toString;
            mentor.updateProfileImage(fileFullPath);
        }

```